arelle/plugin/validate/ESEF/ESEF_Current/ValidateXbrlFinally.py

```diff
@@ -26,6 +26,7 @@
 from arelle.ModelValue import qname
 from arelle.ModelXbrl import ModelXbrl
 
+from arelle.plugin.validate.Util import etreeIterWithDepth
```

Comment:
This import path won’t work in frozen builds since plugins are relocated to the root directory. While relative imports are an option, placing a non-plugin module in the plugin directory can lead to other issues. It’s better to move code under `arelle.utils` in Arelle’s core instead if you want to reuse code between plugins.

Reply:
Moved to `arelle.utils.validation.ValidationUtils`